inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -78,367 +78,341 @@ for (pkg in required_packages) {
 
-## define clean variable nams using clean_labels from the epitrix package. This
-## function defines rules for variable naming; for example, it changes spaces
-## and dots to "_" and sets all characters to lowercase.
+## Use the function msf_dict_rename_helper() to create a template based on the
+## AJS dictionary. This will copy a rename command like the one above to your
+## clipboard.
 
-# cleaned_colnames <- epitrix::clean_labels(colnames(linelist_raw))
-## overwrite variable names with defined clean names
-# colnames(linelist_cleaned) <- cleaned_colnames
+# msf_dict_rename_helper("AJS")
 
+## Paste the result below and your column names to the matching variable.
+## Be careful! You still need to be aware of what each variable means and what
+## values it takes.
+## If there are any variables that are in the MSF dictionary that are not in
+## your data set, then you should comment them out, but be aware that some
+## analyses may not run because of this. 
 
 
 
-## Renaming variables to match the template -----------------------------------
-## OPTIONAL: below is just an example in case you want to specifically change a
-## few names you can also change specific var names using the *rename*
-## function. In this example, we have the columns "gender" and "age" that we
-## want to rename as "sex" and "age_years".
-## The formula for this is rename(data, NEW_NAME = OLD_NAME). 
+## Here is an EXAMPLE for changing a few specific names. function. In this
+## example, we have the columns "gender" and "age" that we want to rename as
+## "sex" and "age_years". 
+## The formula for this is rename(data, NEW_NAME = OLD_NAME).
 
 # linelist_cleaned <- rename(linelist_cleaned, 
 #                            sex       = gender, # TEXT
-#                            age_years = age  # INTEGER_POSITIVE
+#                            age_years = age     # INTEGER_POSITIVE
 # )
 
-## You can use the function msf_dict_rename_helper() to create a template based
-## on the AJS dictionary. This will copy a rename command like the one above
-## to your clipboard.
-# msf_dict_rename_helper("AJS")
-## Paste the result here and add add the column names from linelist_cleaned
-## to the appropriate variables. PLEASE NOTE: this is not a foolproof solution;
-## you still need to be aware of what each variable means and what values it
-## takes. 
-
-## If there are any variables that are in the MSF dictionary that are not in
-## your data set, then you should comment them out, but be aware that some
-## analyses may not run because of this. 
-
 ```
 
 
 
 
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+read_population_data
 
+This template uses population data to calculate things like incidence.
 
+There are two options:
+- You can read in a spreadsheet with age group and region data. (You can see an
+example spreadsheet TODO.)
+- If you have the total or regional populations, you can estimate the age group
```

Comment:
can also type in counts now...

Reply:
can you give an example? or do i just need to rebuild pkg to get the correct help file?

Reply:
think merge master and then build for correct help file